### ISSUE TITLE
well my game cant start after i downloaded it

### ISSUE THREAD

**Issue by keyvanlp** [OpenTTD/OpenTTD#8979]
## Version of OpenTTD
1.11.0 (0)

## Expected result
game starting

## Actual result
error screen shows up

## Steps to reproduce
downloading the game and starting it

## Crash Report

```
*** OpenTTD Crash Report ***

Crash at: Fri Apr  9 15:58:17 2021
In game date: 2095-05-19 (20)

Crash reason:
 Exception: C0000005
 Location:  00007FF6067435FC
 Message:   <none>

OpenTTD version:
 Version:    1.11.0 (0)
 NewGRF ver: 1b086d64
 Bits:       64
 Endian:     little
 Dedicated:  no
 Build date: Apr  1 2021 12:39:23

Registers:
 RAX: 000002AECC153000 RBX: 0000000000005A60 RCX: 0000000000000000 RDX: 0000000000000000
 RSI: 0000000000000000 RDI: 0000000000000000 RBP: 0000000000000000 RSP: 000000CEA7CFF340
 R8:  0000000000000000 R9:  0000000000000000 R10: 00000000000002D3 R11: 000000CEA7CFF408
 R12: 000002AEC3786710 R13: 00007FF606ACE300 R14: 00007FF60698DA70 R15: FFFFFFFFFFFFFFFF
 RIP: 00007FF6067435FC EFLAGS: 00010206

 Bytes at instruction pointer:
 40 38 7B 18 74 14 4C 8B CE 4C 8B C3 40 0F B6 D7 41 8B CA E8 8C FE FF FF

Stack trace:
 FFFFFFFF 00000000 C37C1F40 000002AE 0698DA70 00007FF6 00000004 00000000
 00000001 00000000 00000000 00000000 C48B80E0 000002AE 06590581 00007FF6
 00000000 00000000 00000000 00000000 00000000 00000000 FFFFFFFF FFFFFFFF
 C37C1F40 000002AE C37C1F40 000002AE C48B80E0 000002AE 06590490 00007FF6
 A7CFF5B0 000000CE 00000000 000002AE C37C1F40 000002AE C37C1F40 000002AE
 FFFFFFFF FFFFFFFF 0698DA70 00007FF6 00000000 00000000 C37C1F40 000002AE
 C37C1F40 000002AE 065988ED 00007FF6 CC021DE0 000002AE 00000000 000002AE
 A7CFF5B0 000000CE C3786710 000002AE 0000000A 000002AE 7D8F148C 00000008
 C37C1F80 000002AE 0659AC28 00007FF6 C37C1FD0 000002AE CC1532B0 000002AE
 C48B39C0 000002AE C37C1FD0 000002AE 00000000 00000000 0659739F 00007FF6
 C37C1FD0 000002AE C37C1F80 000002AE C37C1F40 000002AE 00000000 00000000
 C37ACB00 000002AE 066D82D9 00007FF6 C37ACB00 000002AE C37ACB00 000002AE
 00000000 00000000 00000000 00000000 00000000 00000000 0C99DB17 0000D28D
 A7CFF460 000000CE 00000000 00000000 0000000A 00000000 A7CFF440 000000CE
 00000000 00000001 00000000 00000000 00000000 00000000 00000000 00000000
 00000000 00007FFA A7CFF9A8 000000CE 0694D2F0 00007FF6 00000000 00000000
 00000000 00000000 A7CFF9A0 000000CE 00000000 00000000 065045E4 00007FF6
 0000027F 00000000 00000101 00000000 00000000 00000000 00001FA0 0002FFFF
 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000
 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000
 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000
 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000
 00000000 00000000 00000000 00000000 00000000 00000000 23899AA0 00007FFA
 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000

Operating system:
 Name:     Windows
 Release:  10.0.19042 ()
 Compiler: MSVC 1928

Configuration:
 Blitter:      40bpp-anim
 Graphics set: OpenGFX (7761)
 Language:     E:\steam system\steamapps\common\OpenTTD\lang\german.lng
 Music driver: dmusic
 Music set:    OpenMSX (7761)
 Network:      no
 Sound driver: xaudio2
 Sound set:    OpenSFX (7758)
 Video driver: win32-opengl

Fonts:
 Small:  sprite
 Medium: sprite
 Large:  sprite
 Mono:   sprite

AI Configuration (local: 0) (current: 0):
  0: Human
  1: Human
  2: Human
  3: Human
  4: Human
  5: Human

Libraries:
 LZMA:       5.2.5
 LZO:        2.10
 PNG:        1.6.37
 Zlib:       1.2.11

Module information:
 E:\steam system\steamapps\common\OpenTTD\openttd.exe handle: 00007FF606310000 size: 8542208 crc: EE6C3614 date: 2021-04-09 15:40:53
 C:\Windows\SYSTEM32\ntdll.dll handle: 00007FFA25F70000 size: 2024224 crc: 686071F5 date: 2021-03-19 11:05:21
 C:\Windows\System32\KERNEL32.DLL handle: 00007FFA25990000 size: 764976 crc: 721FEC31 date: 2021-03-05 17:18:41
 C:\Windows\System32\KERNELBASE.dll handle: 00007FFA23830000 size: 2922392 crc: E81AFBF5 date: 2021-03-05 17:20:05
 C:\Windows\System32\WS2_32.dll handle: 00007FFA24C20000 size: 427200 crc: 6E37A637 date: 2020-10-17 13:14:38
 C:\Windows\System32\RPCRT4.dll handle: 00007FFA25690000 size: 1222056 crc: 44DB9CA6 date: 2021-03-05 17:20:05
 C:\Windows\System32\IMM32.dll handle: 00007FFA25660000 size: 185448 crc: 36F61BF6 date: 2020-10-17 13:14:39
 C:\Windows\System32\USER32.dll handle: 00007FFA25CA0000 size: 1702416 crc: CCD30A7B date: 2021-03-05 17:19:32
 C:\Windows\System32\win32u.dll handle: 00007FFA23B00000 size: 132760 crc: 40694227 date: 2021-03-19 11:04:53
 C:\Windows\SYSTEM32\OPENGL32.dll handle: 00007FF9FAA50000 size: 1064448 crc: 77B8A2FB date: 2021-03-19 11:06:15
 C:\Windows\System32\GDI32.dll handle: 00007FFA25570000 size: 161680 crc: 2A141BAA date: 2021-03-05 17:19:00
 C:\Windows\System32\msvcrt.dll handle: 00007FFA25C00000 size: 637360 crc: 072737D9 date: 2020-10-17 13:14:37
 C:\Windows\System32\gdi32full.dll handle: 00007FFA23EC0000 size: 1075864 crc: 129DC3C4 date: 2021-03-05 17:20:12
 C:\Windows\System32\msvcp_win.dll handle: 00007FFA23730000 size: 634760 crc: E6732639 date: 2021-03-05 17:20:00
 C:\Windows\SYSTEM32\WINMM.dll handle: 00007FFA14EE0000 size: 147200 crc: 18D40339 date: 2020-10-17 13:13:22
 C:\Windows\System32\ADVAPI32.dll handle: 00007FFA255B0000 size: 689024 crc: 355A9B19 date: 2020-11-27 14:23:40
 C:\Windows\System32\ucrtbase.dll handle: 00007FFA23B30000 size: 1044888 crc: D3CA62A3 date: 2021-03-05 17:20:00
 C:\Windows\System32\sechost.dll handle: 00007FFA25B60000 size: 628616 crc: 4ED65EE2 date: 2021-03-05 17:20:05
 C:\Windows\System32\SHELL32.dll handle: 00007FFA24E20000 size: 7639536 crc: 41DE241A date: 2021-03-19 11:05:36
 C:\Windows\System32\combase.dll handle: 00007FFA245F0000 size: 3507000 crc: 54E9FC3C date: 2021-03-19 11:05:16
 C:\Windows\System32\ole32.dll handle: 00007FFA24A10000 size: 1212232 crc: 0FD6B05F date: 2021-03-05 17:19:54
 C:\Windows\SYSTEM32\USP10.dll handle: 00007FFA1C250000 size: 79360 crc: 9A7F7CC2 date: 2020-10-17 13:14:39
 C:\Windows\SYSTEM32\GLU32.dll handle: 00007FFA01920000 size: 164352 crc: DA0CBD61 date: 2021-03-19 11:06:15
 E:\steam system\gameoverlayrenderer64.dll handle: 00007FF9F5670000 size: 1808104 crc: 4E14C8C5 date: 2021-03-23 21:22:56
 C:\Windows\System32\PSAPI.DLL handle: 00007FFA255A0000 size: 19144 crc: 6E06CEEB date: 2020-10-17 13:14:25
 C:\Windows\SYSTEM32\windows.storage.dll handle: 00007FFA21830000 size: 7965496 crc: C5483783 date: 2021-03-19 11:04:52
 C:\Windows\SYSTEM32\Wldp.dll handle: 00007FFA23120000 size: 173016 crc: C5218D0B date: 2021-01-02 13:34:06
 C:\Windows\System32\SHCORE.dll handle: 00007FFA24960000 size: 704008 crc: 0812F61E date: 2021-03-05 17:19:35
 C:\Windows\System32\shlwapi.dll handle: 00007FFA25E40000 size: 342440 crc: 5693748E date: 2021-03-05 17:20:34
 C:\Windows\system32\uxtheme.dll handle: 00007FFA21170000 size: 627712 crc: A05B5F55 date: 2021-03-05 17:20:19
 C:\Windows\System32\MSCTF.dll handle: 00007FFA24D00000 size: 1129056 crc: 02E7319C date: 2021-03-19 11:05:22
 C:\Windows\System32\OLEAUT32.dll handle: 00007FFA25840000 size: 831024 crc: 8304FB12 date: 2021-03-05 17:20:01
 C:\Windows\SYSTEM32\kernel.appcore.dll handle: 00007FFA21630000 size: 60464 crc: 7039ACC9 date: 2020-10-17 13:14:15
 C:\Windows\System32\bcryptPrimitives.dll handle: 000002AEC4E60000 size: 523200 crc: 1A4DFB8D date: 2021-01-02 13:34:14
 C:\Windows\System32\clbcatq.dll handle: 00007FFA25AB0000 size: 672976 crc: AD4CB4A3 date: 2020-10-17 13:14:26
 C:\Windows\SYSTEM32\atig6pxx.dll handle: 00007FFA0EB40000 size: 17920 crc: B6E883A9 date: 2015-01-13 14:20:52
 C:\Windows\SYSTEM32\atio6axx.dll handle: 0000000069030000 size: 26017280 crc: 699A202A date: 2015-01-13 14:46:26
 C:\Windows\System32\SETUPAPI.dll handle: 00007FFA24180000 size: 4650576 crc: F65AC9E1 date: 2021-03-19 11:05:34
 C:\Windows\System32\cfgmgr32.dll handle: 00007FFA23E40000 size: 305272 crc: DE2280BE date: 2020-10-17 13:14:36
 C:\Windows\System32\bcrypt.dll handle: 00007FFA23E90000 size: 146240 crc: 50F3C5E2 date: 2021-01-02 13:34:06
 C:\Windows\SYSTEM32\VERSION.dll handle: 00007FFA192D0000 size: 31496 crc: 546E295F date: 2020-10-17 13:14:49
 C:\Windows\SYSTEM32\dwmapi.dll handle: 00007FFA21340000 size: 168408 crc: BE283206 date: 2021-03-05 17:19:55
 C:\Windows\SYSTEM32\atiadlxx.dll handle: 00007FFA17050000 size: 535552 crc: A22303DE date: 2015-01-13 14:20:58
 C:\Windows\SYSTEM32\USERENV.dll handle: 00007FFA235B0000 size: 176656 crc: 82C8FFDF date: 2020-10-17 13:14:29
 C:\Windows\SYSTEM32\WSOCK32.dll handle: 00007FFA17040000 size: 18944 crc: F78C7419 date: 2019-12-07 09:08:41
 C:\Windows\SYSTEM32\WTSAPI32.dll handle: 00007FFA1F2B0000 size: 67192 crc: 7604358D date: 2020-10-17 13:13:31
 C:\Windows\SYSTEM32\DEVOBJ.dll handle: 00007FFA23460000 size: 165832 crc: 9AB2CB9F date: 2020-10-17 13:14:36
 C:\Windows\System32\WINTRUST.dll handle: 00007FFA237D0000 size: 382720 crc: 255FF0BB date: 2021-03-05 17:19:21
 C:\Windows\System32\CRYPT32.dll handle: 00007FFA23CE0000 size: 1425440 crc: 71673E8A date: 2021-03-19 11:05:16
 C:\Windows\SYSTEM32\MSASN1.dll handle: 00007FFA232A0000 size: 64072 crc: FDFACF86 date: 2020-10-17 13:14:28
 C:\Windows\SYSTEM32\aticfx64.dll handle: 00007FFA19560000 size: 1094024 crc: DAF97EC9 date: 2015-01-13 15:45:56
 C:\Windows\SYSTEM32\atig6txx.dll handle: 00007FFA0E600000 size: 41984 crc: FEBD1D47 date: 2015-01-13 14:20:50
 C:\Windows\SYSTEM32\textinputframework.dll handle: 00007FFA19460000 size: 1021144 crc: C1B80C04 date: 2021-03-19 11:04:50
 C:\Windows\SYSTEM32\CoreUIComponents.dll handle: 00007FFA20930000 size: 3537520 crc: 78335519 date: 2020-10-17 13:14:01
 C:\Windows\SYSTEM32\CoreMessaging.dll handle: 00007FFA20D70000 size: 986464 crc: EDF107E3 date: 2021-03-05 17:19:59
 C:\Windows\SYSTEM32\ntmarta.dll handle: 00007FFA22400000 size: 191656 crc: B2A51DA2 date: 2020-10-17 13:14:29
 C:\Windows\SYSTEM32\wintypes.dll handle: 00007FFA20260000 size: 1393984 crc: DE0F60F1 date: 2021-03-19 11:05:16
 C:\Windows\System32\MMDevApi.dll handle: 00007FFA16BD0000 size: 529424 crc: DFC9B2A0 date: 2021-03-05 17:18:20
 C:\Windows\SYSTEM32\AUDIOSES.DLL handle: 00007FFA146E0000 size: 1578512 crc: 035CD2FE date: 2021-03-05 17:18:21
 C:\Windows\SYSTEM32\powrprof.dll handle: 00007FFA22CB0000 size: 295464 crc: 74230007 date: 2020-10-17 13:14:29
 C:\Windows\SYSTEM32\UMPDC.dll handle: 00007FFA22B20000 size: 64552 crc: B19D9CE1 date: 2020-10-17 13:13:23
 C:\Windows\System32\PROPSYS.dll handle: 00007FFA1F0F0000 size: 1009232 crc: A53AC761 date: 2021-03-19 11:04:52
 C:\Windows\SYSTEM32\dxgi.dll handle: 00007FFA22000000 size: 979792 crc: 7C69B360 date: 2021-03-05 17:19:00
 C:\Windows\SYSTEM32\d3d11.dll handle: 00007FFA1FA30000 size: 2503504 crc: D4639E70 date: 2021-03-05 17:19:00
 C:\Windows\SYSTEM32\atiuxp64.dll handle: 00007FFA192E0000 size: 61464 crc: 1949A53C date: 2015-01-13 15:46:36
 C:\Windows\SYSTEM32\apphelp.dll handle: 00007FFA20FA0000 size: 570880 crc: 0FE4DB70 date: 2020-10-17 13:14:25
 C:\Windows\SYSTEM32\AcGenral.DLL handle: 00007FFA18B30000 size: 370176 crc: 4320519B date: 2020-10-17 13:15:49
 C:\Windows\SYSTEM32\SspiCli.dll handle: 00007FFA23570000 size: 230904 crc: DED4F11C date: 2021-01-02 13:34:13
 C:\Windows\SYSTEM32\MPR.dll handle: 00007FFA18AE0000 size: 109504 crc: 92B7C454 date: 2020-10-17 13:14:39
 C:\Windows\SYSTEM32\atidxx64.dll handle: 00007FFA18B90000 size: 7569128 crc: 98A43701 date: 2015-01-13 15:45:48
 C:\Windows\SYSTEM32\dxcore.dll handle: 00007FFA1A0F0000 size: 219016 crc: 72966353 date: 2020-10-17 13:13:37
 C:\Windows\system32\mswsock.dll handle: 00007FFA22EA0000 size: 418416 crc: 7609CC3B date: 2020-10-17 13:14:28
 C:\Windows\SYSTEM32\xaudio2_8.dll handle: 000002AEC51C0000 size: 3584 crc: BC198863 date: 2019-12-07 09:08:07
 C:\Windows\SYSTEM32\xaudio2_9.DLL handle: 00007FF9FD640000 size: 623616 crc: BE3685CE date: 2020-11-27 14:23:39
 C:\Windows\SYSTEM32\AVRT.dll handle: 00007FFA1D400000 size: 30480 crc: D6790ED2 date: 2020-10-17 13:13:22
 C:\Windows\SYSTEM32\resourcepolicyclient.dll handle: 00007FFA21250000 size: 71784 crc: 38C47F6A date: 2020-10-17 13:13:36
 C:\Windows\System32\dmusic.dll handle: 00007FFA018F0000 size: 136192 crc: BDA2573A date: 2019-12-07 09:09:37
 C:\Windows\SYSTEM32\winmmbase.dll handle: 00007FFA0E7C0000 size: 144592 crc: B331D5DD date: 2019-12-07 09:08:07
 C:\Windows\System32\DSOUND.dll handle: 00007FF9FC810000 size: 615424 crc: 1C78A884 date: 2019-12-07 09:08:07
 C:\Windows\System32\ksuser.dll handle: 00007FFA1A410000 size: 23264 crc: A73C3DE9 date: 2019-12-07 09:08:07
 C:\Windows\System32\MSACM32.dll handle: 00007FFA1EC90000 size: 112080 crc: 7C3515A4 date: 2019-12-07 09:08:07
 C:\Windows\SYSTEM32\wdmaud.drv handle: 00007FFA126E0000 size: 259584 crc: DB966BEF date: 2019-12-07 09:08:07
 C:\Windows\SYSTEM32\msacm32.drv handle: 00007FFA1A420000 size: 30208 crc: 414EBD44 date: 2020-09-13 09:38:17
 C:\Windows\SYSTEM32\midimap.dll handle: 00007FFA14990000 size: 26624 crc: C81948B1 date: 2020-09-13 09:38:17
 C:\Windows\System32\dmsynth.dll handle: 00007FFA00430000 size: 126464 crc: 2A50C1AD date: 2019-12-07 09:09:37
 C:\Windows\System32\Windows.UI.dll handle: 00007FFA19670000 size: 1317544 crc: C59FEC4F date: 2021-03-05 17:19:46
 C:\Windows\System32\InputHost.dll handle: 00007FFA19300000 size: 1379072 crc: 6468F174 date: 2021-03-05 17:19:12
 C:\Windows\System32\WindowManagementAPI.dll handle: 00007FFA1C1A0000 size: 643072 crc: 4D2F7971 date: 2021-03-05 17:19:15
 C:\Windows\System32\twinapi.appcore.dll handle: 00007FFA1BF50000 size: 2100112 crc: 607D6B93 date: 2021-03-05 17:18:55

---- gamelog start ----
Tick 0: new game started
Revision text changed to 1.10.3, savegame version 218, not modified, _openttd_newgrf_version = 0x1a086d64
New game mode: 1 landscape: 2
Tick 1280: game loaded
New game mode: 2 landscape: 2
Tick 1280: game loaded
New game mode: 1 landscape: 2
Tick 39224: game loaded
New game mode: 2 landscape: 2
Tick 39224: game loaded
New game mode: 1 landscape: 2
Tick 60493: settings changed
Setting changed: vehicle.never_expire_vehicles : 0 -> 1
Tick 52147: game loaded
New game mode: 2 landscape: 2
Tick 52147: game loaded
New game mode: 1 landscape: 2
Tick 39493: game loaded
New game mode: 2 landscape: 2
Tick 39493: game loaded
New game mode: 1 landscape: 2
Tick 29222: game loaded
New game mode: 2 landscape: 2
Tick 29222: game loaded
New game mode: 1 landscape: 2
Tick 29222: game loaded
Revision text changed to 1.11.0, savegame version 290, not modified, _openttd_newgrf_version = 0x1b086d64
New game mode: 0 landscape: 2
---- gamelog end ----

Recent news messages:

*** End of OpenTTD Crash Report ***

Decoded stack trace:
[00] openttd              0x00007FF6067435FC
[01] openttd              0x00007FF606590581
[02] openttd              0x00007FF6065988ED
[03] openttd              0x00007FF60659AC28
[04] openttd              0x00007FF60659739F
[05] openttd              0x00007FF6066D82D9
[06] openttd              0x00007FF60650F568
[07] openttd              0x00007FF606876162
[08] KERNEL32             0x00007FFA259A7034 BaseThreadInitThunk + 20
[09] ntdll                0x00007FFA25FC2651 RtlUserThreadStart + 33

*** End of additional info ***
```

**Comment by glx22** [comment]
Please also attach `crash.dmp` (in a zip)

**Comment by LordAro** [comment]
And ideally, some details about your hardware - in particular, GPU(s) & graphics driver details, and/or running OTTD with `-ddriver=4` flag on start up

**Comment by LordAro** [comment]
Likely duplicate of #8967. Should be fixed in 1.11.2, released now!
